VocabFlashcard/src/client.tsx

```
@@ -0,0 +1,239 @@
+          <div className={styles.wordContainer}>
+            <Grid container justifyContent="center" alignItems="center" spacing={2}>
+              <Grid display="flex" justifyContent="center" alignItems="center" size={12}>
+                <Card>{card("English", word.english, true)}</Card>
+              </Grid>
+              <Grid size={12}>
+            <Typography variant="h6" color="white" align="center">
+              Choose one from below options...
+            </Typography>
+            </Grid>
+              {options.map((option, index) => (
+                <Grid size={{ xs: 6, md: 6 }} display="flex" justifyContent="center" alignItems="center">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)